core/domain/classifier_services.py

```diff
@@ -153,6 +154,49 @@ def handle_non_retrainable_states(exploration, state_names, exp_versions_diff):
         job_exploration_mappings)
 
 
+def convert_strings_to_float_numbers_in_classifier_data(classifier_data):
+    """Converts all floating point numbers in classifier data to string.
+
+    The following function iterates through entire classifier data and converts
+    all string values which are successfully matched by regex of floating point
+    numbers to corresponding float values.
+
+    Args:
+        classifier_data: dict|list|string|int. The original classifier
+            data which needs conversion of floats from strings to floats.
+
+    Raises:
+        Exception. If classifier data contains an object whose type is other
+            than integer, string, dict or list.
+
+    Returns:
+        dict|list|string|int. Original classifier data dict with float values
+            converted back from string to float.
+    """
+    if isinstance(classifier_data, dict):
```

Comment:
Rename to classifier_data_with_floats_stringified and create a new classifier_data variable to return

Reply:
Done